code/modules/mob/mob.dm

```diff
@@ -866,7 +866,10 @@ var/list/slot_equipment_priority = list( \
 /mob/MouseDrop(mob/M as mob)
 	..()
 	if(M != usr) return
-	if(M.small) return // Stops pAI drones and small mobs (borers, parrots, crabs) from stripping people. --DZD
+	if(isliving(M)) // Ewww
```

Comment:
This is gross, but unfortunately you can't do an override of this for living without causing horrific runtimes and breaking `MouseDrop` everywhere.